portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,72 @@
 #content {
+  margin-top: 100px;
   margin-left: auto;
   margin-right: auto;
-  width: 650px;
+  width: 800px; /*650 */
 }
-
 #greeting-container {
-  margin-top: 20px;
+  margin-top: 50px;
+}
+h1 {
+    color:white;
```

Comment:
In general, it's preferable to pick a style for the code and be consistent. For example, after the `:`, you mostly have a space, but not here. The Google style guide does use a space.